addons/web_editor/static/src/js/rte.js

```diff
@@ -533,9 +533,8 @@ var RTE = Widget.extend({
             }, 150); // setTimeout to remove flickering when change to editable zone (re-create an editor)
             this.$last = null;
         }
-        if ($editable.length && (!this.$last || this.$last[0] !== $editable[0]) &&
-                ($target.closest('[contenteditable]').attr('contenteditable') || "").toLowerCase() !== 'false') {
-
+        if (($editable.length && ($target.closest('[contenteditable]').attr('contenteditable') || "").toLowerCase() !== 'false' || dom.isImg($target[0]))
+            && (!this.$last || this.$last[0] !== $editable[0])) {
```

Comment:
Hello @qsm-odoo , It seems like a generic issue ( see : https://youtu.be/qh2xUR_kM-E ). If we drag and drop any snippet having contenteditable false, image is not selectable / clickable by default. Once we save the page, behavior is proper (image options are enabled). Should we still fix it withing slideshow snippet ?